### ISSUE TITLE
Rename about-us page to about

### ISSUE THREAD

**Issue by drubgrubby** [hackforla/website#864]
### Dependency
~#966 #967 #862~

### Overview
We need to rename the page to from about-us to "about" to maintain consistent naming convention throughout the project.

### Action Items

- [x] There is an _about.scss file.  Figure out if is it still being used for anything, and if it can be renamed, deleted, etc. 
- [x] Rename the about-us page to about
- [x] Rename _about-us.scss to _about.scss (unless the current _about.scss can't be renamed for some reason)
- [x] Rename assets/js/about-us.js to about.js . 

### Resources/Instructions
https://www.hackforla.org/about-us

**Comment by drubgrubby** [comment]
@evancchen - Hi Evan.  I'm going to unasign you from this issue and put it back in the icebox.  There are dependencies on this issue that need to be completed before we do it.  You can see them listed at the top of the issue.  If you can find an issue from the prioritized backlog, that is best.  If you are having trouble finding an issue, let me or Josh know.

**Comment by akibrhast** [comment]
@drubgrubby Updated one of the dependency, since that issue is now closed

**Comment by akibrhast** [comment]
@drubgrubby Do you think this is something that could be worked on at the moment given all the previously listed dependencies are closed ?

There are quite a few open issues that needs to be done on about us. Should this one be taken care of first as a priority?

#1089 #1064 #1063 #863 #861 #860

While none of those are dependent on this and this is not dependent on any of those.

Getting the renaming/deleting of  the aforementioned files out of the way early might be good idea , since that way anyone working on those above mentioned issues and new issue that might come along in the future will all have a standardized version.

I fear that waiting on this is going to keep pushing it back constantly.. and at some point somewhere going to cause merge conflicts.

**Comment by drubgrubby** [comment]
@akibrhast - This is probably as good a time as any.  The only person who is, I think, working on any of these at the moment is @kevinreber , who is working on #863 .  I'm tagging him here so he knows this is happening.  If you need, you can work together to resolve any conflicts with the file name change.

**Comment by kevinreber** [comment]
I'm currently just waiting for some styling feedback from @daniellex0. I can create a PR of what I have now to avoid any possible merge conflicts and re-visit the styling in a separate issue.

**Comment by drubgrubby** [comment]
@akibrhast - (if you haven't already) Can you work out with Kevin if you think he should push what he has now and then do the other changes after the rename, or if you want to wait to push the rename, or some third solution?

**Comment by daniellex0** [comment]
@akibrhast @drubgrubby Sorry about the delay, I was just able to get back to @kevinreber - however the edits I sent to him aren't super important, so he can push the changes and make the edits after

**Comment by kevinreber** [comment]
### Progress
Trying to find the best way to manage the styling in `_about.scss`
### Blockers
The styling in `_about.scss` doesn't apply to the about page, but is used for various elements on the website(screenshots below). I'm trying to find the best way to re-organize the styles in `_about.scss` and was wondering if anyone has any good ideas?
![image](https://user-images.githubusercontent.com/42260999/109428049-0ded7880-79aa-11eb-8f36-30fed8e47ea6.png)
![image](https://user-images.githubusercontent.com/42260999/109428074-22ca0c00-79aa-11eb-92cd-f703126faa49.png)

### Availability
Afternoon - Evening

### ETA 
Next Meeting

**Comment by kevinreber** [comment]
### Additional Action Items
Regarding `_about.scss` and `_about-us.scss` files
- [x] Take everything from `_about.scss` and paste into a new file `_home.scss` 
- [x] Take everything from `_about-us.scss` and paste into `_about.scss`
- [x] Delete `_about-us.scss` and remove from `main.scss`
- [x] Make sure `_home.scss` is imported and compiled in `main.scss`
